packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```diff
@@ -33,18 +33,18 @@
   "ip_alerts_service_other": "{} alertes pour ce service",
   "ip_alerts_spam": "Aucune IP bloquée (Antispam)",
   "ip_alerts_spam_1": "1 IP bloquée (Antispam)",
-  "ip_alerts_spam_help": "Spam (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
+  "ip_alerts_spam_help": "<strong>Spam</strong> (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
   "ip_alerts_spam_other": "{} IPs bloquées (Antispam)",
   "ip_alerts_antihack": "Aucune IP bloquée (Anti-hack)",
   "ip_alerts_antihack_1": "1 IP bloquée (Anti-hack)",
-  "ip_alerts_antihack_help": "Anti-hack (blocage du service suite à une suspicion d’activité malveillante)",
+  "ip_alerts_antihack_help": "<strong>Anti-hack</strong> (blocage du service suite à une suspicion d’activité malveillante)",
   "ip_alerts_antihack_other": "{} IPs bloquées (Anti-hack)",
   "ip_alerts_arp": "Aucune IP bloquée (ARP)",
   "ip_alerts_arp_1": "1 IP bloquée (ARP)",
   "ip_alerts_arp_other": "{} IPs bloquées (ARP)",
   "ip_alerts_mitigation": "Aucune IP attaquée (Mitigation)",
   "ip_alerts_mitigation_1": "1 IP attaquée (Mitigation)",
-  "ip_alerts_mitigation_help": "Mitigation d’attaque DDoS (protection du service subissant une attaque DDoS)",
+  "ip_alerts_mitigation_help": "<strong>Mitigation d’attaque DDoS</strong> (protection du service subissant une attaque DDoS)",
```

Comment:
**question:** Can we split into 2 sentences and avoid HTML In translation files.